### PR TITLE
Feature/profile closet

### PR DESCRIPTION
**Pull request by fredbordel**
Created 2 new functions in useApplicationData: 1) Removes item from the cart. 2) Clear the cart.
On cart component, you'll see temporary delete buttons attached to each item. We can reuse this functionality later in a different form.

Added top and bottom wave in almost every page expect top wave in Profile.
Added new icons in header of feed-cart-checkout-confirmation and new icons for the filtering form.

### CODE REVIEW

File: client/scss/wave.scss [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
+//   content: "";
+//   display: block;
+//   position: absolute;
+//   border-radius: 2% 100%;
+//   width: 150px;
+//   height: 150px;
+//   background-color: #70774B;
+//   left: -10px;
+//   top: 130px;
+// }
```
Comment (Miaouchkinz) [inline client/scss/wave.scss]: We may as well delete this whole file if it's all commented out now. Don't think we'll be using it anymore right?

File: client/scss/index.scss [hunk 2]
```diff
@@ -1,10 +1,11 @@
 @import "./button";
 @import "./text-input.scss";
-@import "./Feed.scss";
+@import "./Feed";
```
Comment (Miaouchkinz) [inline client/scss/index.scss]: Curious why this one and the import for button don't have .scss? Was that prettier? 🤔

File: client/src/components/Cart.jsx [hunk 3]
```diff
@@ -2,12 +2,15 @@ import React from "react";
 import { Link } from "react-router-dom";
 import Icon from "./common/iconButton";
 
-export default function Cart({ cart, clothing }) {
+export default function Cart({ cart, clothing, removeFromCart }) {
   const itemInCartId = function() {
```
Comment (Miaouchkinz) [inline client/src/components/Cart.jsx]: is this for all of the items? If so maybe change the variable name to something like: itemsInCartId or cartItemsIDList?

File: client/src/components/Cart.jsx [hunk 4]
```diff
@@ -26,7 +29,14 @@ export default function Cart({ cart, clothing }) {
           }
           id={clothingItem.size}
         ></img>
-        <footer></footer>
+        <div>
+          <img
+            alt="Remove item from cart button."
```
Comment (Miaouchkinz) [inline client/src/components/Cart.jsx]: look at those beautiful IMG ALTS 😍

File: client/src/components/Cart.jsx [hunk 5]
```diff
@@ -60,21 +70,36 @@ export default function Cart({ cart, clothing }) {
...
-        Selected items <span className="total_item_in_cart">{cart.length}</span>
-      </h1>
-      <div className="clothingItem_cart_carousel">
-        {checkIfCartEmpty(cartItemList)}
-      </div>
-      <div>
-        <h1>Future machine learning cool stuff here</h1>
+      <div className="content_container">
+        <h1>
+          Selected items{" "}
```
Comment (Miaouchkinz) [inline client/src/components/Cart.jsx]: Guessing that the {" "} gets filled in as part of another function?

File: client/src/components/Closet.jsx [hunk 6]
```diff
@@ -1,7 +1,30 @@
 import React from "react";
 
-export default function Closet(props) {
+export default function Closet({ allClothing, userId, userName }) {
+  const itemsOfCurrentUser = (clothingList, userIdentity) => {
```
Comment (Miaouchkinz) [inline client/src/components/Closet.jsx]: what's userIdentity?

File: client/src/components/Feed.jsx [hunk 7]
```diff
@@ -41,38 +35,60 @@ export default function Feed({ clothing, clothingCategories, cart, setCart }) {
     }
   };
 
-  // Creates array of labels for each category from the allCategories object
```
Comment (Miaouchkinz) [inline client/src/components/Feed.jsx]: Did you mean to take out all your comments? Just wondering if they were accidentally removed when you were trying to pull it out into helper files?

File: client/src/components/Feed.jsx [hunk 8]
```diff
@@ -41,38 +35,60 @@ export default function Feed({ clothing, clothingCategories, cart, setCart }) {
...
               key={allCategories[category].id}
               name={allCategories[category].id}
               type="checkbox"
               checked={activeCategories[name]}
               onChange={() => ifCheckBoxActive(category, allCategories)}
             />
-            {allCategories[category].name}
+            <div className="size_icon">
+              <img
+                alt="Icon for filtering clothing."
```
Comment (Miaouchkinz) [inline client/src/components/Feed.jsx]: add "by size"

File: client/src/components/Feed.jsx [hunk 9]
```diff
@@ -91,23 +107,19 @@ export default function Feed({ clothing, clothingCategories, cart, setCart }) {
       );
     }
   };
-  ///////////////////////////////////////////////// ///////////////////////////
-  // Variable that holds all clothing (where available = true) from backend
+
   const clothingList = clothing && clothing;
```
Comment (Miaouchkinz) [inline client/src/components/Feed.jsx]: What does this do?

File: client/src/components/Profile.jsx [hunk 10]
```diff
@@ -1,56 +1,82 @@
...
+      })
+      .catch(err => console.log("logout err", err));
+  };
 
   return (
     <div id="profile-page">
-      <div id="wave">
-        <div id="inner-wave"></div>
-      </div>
+      <div id="wave">{/* <div id="inner-wave"></div> */}</div>
```
Comment (Miaouchkinz) [inline client/src/components/Profile.jsx]: remove the commented out inner wave if we're no longer using this styling :)

File: client/scss/wave.scss (hunk 1 above)
Comment (fredbordel) [reply client/scss/wave.scss]: All your artwork :((((( ahaah

File: client/scss/index.scss (hunk 2 above)
Comment (fredbordel) [reply client/scss/index.scss]: I don't think so.... Let's add back those scss!

File: client/src/components/Cart.jsx (hunk 3 above)
Comment (fredbordel) [reply client/src/components/Cart.jsx]: itemsInCartId will win!

File: client/scss/index.scss (hunk 2 above)
Comment (Miaouchkinz) [reply client/scss/index.scss]: hahah it's okay, we have a better solution now 👯

File: client/src/components/Cart.jsx (hunk 3 above)
Comment (fredbordel) [reply client/src/components/Cart.jsx]: It'a a list of all the id's of the items that are in the cart

File: client/src/components/Cart.jsx (hunk 5 above)
Comment (fredbordel) [reply client/src/components/Cart.jsx]: At think it was a default thing that I never used... or i don't remember what is was there for ahaha

File: client/src/components/Closet.jsx (hunk 6 above)
Comment (fredbordel) [reply client/src/components/Closet.jsx]: userId but fancier? ahaha

File: client/src/components/Feed.jsx (hunk 9 above)
Comment (fredbordel) [reply client/src/components/Feed.jsx]: oups, an old syntax used in an old function... I'll just leave clothing

**Pull request merged**